example/src/main/java/com/vimeo/android/networking/example/TestApp.java

```diff
@@ -28,10 +32,32 @@ public void onCreate() {
         super.onCreate();
 
         mContext = this;
-
         AccountPreferenceManager.initializeInstance(mContext);
 
         // <editor-fold desc="Vimeo API Library Initialization">
+        Configuration.Builder configBuilder;
+        // This check is just as for the example. In practice, you'd use one technique or the other.
+        if (ACCESS_TOKEN_PROVIDED) {
+            configBuilder = getAccessTokenBuilder();
+        } else {
+            configBuilder = getIdAndSecretBuilder();
+        }
+        if (IS_DEBUG_BUILD) {
+            // Disable cert pinning if debugging (so we can intercept packets)
+            configBuilder.enableCertPinning(false);
+            configBuilder.setLogLevel(LogLevel.VERBOSE);
+        }
+        VimeoClient.initialize(configBuilder.build());
+        // </editor-fold>
+    }
+
+    public Configuration.Builder getAccessTokenBuilder() {
+        // The values file is left out of git, so you'll have to provide your own access token
+        String accessToken = getString(R.string.access_token);
+        return new Configuration.Builder(Vimeo.VIMEO_BASE_URL_STRING, accessToken);
+    }
+
+    public Configuration.Builder getIdAndSecretBuilder() {
```

Comment:
It's just for the example, but I think definitely makes it more clear - just made the change :+1: